animepahe-dl.sh

```diff
@@ -262,12 +262,13 @@ main() {
     set_args "$@"
     set_var
 
-    if [[ -z "${_ANIME_SLUG:-}" ]]; then
+    if [[ ! -s ./anime.list ]]; then
         download_anime_list
-        [[ ! -s "$_ANIME_LIST_FILE" ]] && print_error "$_ANIME_LIST_FILE not found!"
-        _ANIME_SLUG=$($_FZF < "$_ANIME_LIST_FILE" | awk -F']' '{print $1}' | sed -E 's/^\[//')
     fi
 
+    _ANIME_NAME="${_ANIME_NAME:-}"
+
+    _ANIME_SLUG=$($_FZF --query "$_ANIME_NAME" < "$_ANIME_LIST_FILE" | awk -F']' '{print $1}' | sed -E 's/^\[//')
```

Comment:
1) Add `-1` to output only match in fzf
2) Remove `_ANIME_NAME="${_ANIME_NAME:-}"`, put `"${_ANIME_NAME:-}"` in fzf parameter part like `...$_FZF --query "${_ANIME_NAME:-}"...`
3) Please consider to use another variable name, `_ANIME_NAME` is also used below for other purpose, check line 273
4) Please keep slug usage,  in case it needs to be used in some case without fzf interaction